lib/parsers/csv_layout.rb

```diff
@@ -88,13 +88,13 @@ def parse
       @data ||= @csv_parser.to_a.map do |line|
         next if line.nil? || line.length == 0
```

Comment:
Not sure if "next if" here is generating a nil element in the array that is then being removed by compact at the end

Could leave as is with comment explaining the compact at the end or may be better to pre-filter all blank lines to reduce code complexity.

Also what if all lines are blank? Does this raise an error somewhere?